### PR TITLE
More package.ini

### PR DESCRIPTION
**Pull request by privat**
package.ini file are not mandatory, so to convince people to use then, here is a new tool.

### CODE REVIEW

File: lib/gmp/package.ini [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name=gmp
+tags=maths,lib
+maintainer=PatrickBlanchette <PatrickBlanchette@users.noreply.github.com>
```
Comment (xymus) [inline lib/gmp/package.ini]: A space is missing in "Patrick Blanchette"

File: bin/nit-makepackageini [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# limitations under the License.
+
+# This shell script creates (or override) a package.ini template file in the current directory.
+#
+# Default values are guessed from git and the file system.
+
+name=`basename $PWD`
+maint=`git shortlog -esn . | head -n 1 | sed 's/\s*[0-9]*\s*//'`
+dir=`git rev-parse --show-prefix`
+remote=`git remote get-url origin`
```
Comment (xymus) [inline bin/nit-makepackageini]: My git is too old for get-url...

File: bin/nit-makepackageini [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+[package]
+name=$name
+tags=
+maintainer=$maint
+license=Apache-2.0
+[upstream]
+FIN
+
+case "$remote" in
+	*/nit.nit)
```
Comment (xymus) [inline bin/nit-makepackageini]: I can't test locally but shouldn't this be `nit.git`?

File: bin/nit-makepackageini [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+dir=`git rev-parse --show-prefix`
+remote=`git remote get-url origin`
+root=${remote%.git}
+
+cat > package.ini <<FIN
+[package]
+name=$name
+tags=
+maintainer=$maint
+license=Apache-2.0
```
Comment (xymus) [inline bin/nit-makepackageini]: I would make Apache-2.0 the default only in the Nit repo. We don't want people to publish their software under this license without their full consent.

File: examples/calculator/package.ini [hunk 5]
```diff
@@ -9,4 +9,4 @@ git=https://github.com/nitlang/nit.git
 git.directory=examples/calculator/
 homepage=http://nitlanguage.org
 issues=https://github.com/nitlang/nit/issues
-apk=http://nitlanguage.org/fdroid/apk/calculator21.apk
+apk=http://nitlanguage.org/fdroid/apk/calculator.apk
```
Comment (privat) [inline examples/calculator/package.ini]: si l'app s'appelle org.nitlanguage.calculator fdroid fabrique des fichier du genre org.nitlanguage.calculator_1705090921.apk

mon script ramasse ces trucs et crée des fichiers avec des noms cours comme calculator.apk
TLDR si tu veux calculator21.apk il faudait que le nom de l'app soit org.nitlanguage.calculator21

File: lib/gmp/package.ini (hunk 1 above)
Comment (privat) [inline lib/gmp/package.ini]: The culprit is @PatrickBlanchette that have a bad git.name configuration (and a notsogood email address).

File: bin/nit-makepackageini (hunk 2 above)
Comment (privat) [inline bin/nit-makepackageini]: https://git-scm.com/docs/git-remote/2.7.0

File: bin/nit-makepackageini (hunk 4 above)
Comment (privat) [inline bin/nit-makepackageini]: make sense

File: bin/nit-makepackageini (hunk 3 above)
Comment (privat) [inline bin/nit-makepackageini]: my origin is asimov:git/nit.git so I needed something general enough :)

Comment (xymus) [inline bin/nit-makepackageini]: So `s/nit.nit/nit.git/`? :P

Comment (privat) [inline bin/nit-makepackageini]: oops. I see your point now

File: bin/nit-makepackageini [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+#
+# Default values are guessed from git and the file system.
+
+name=`basename $PWD`
+maint=`git shortlog -esn . | head -n 1 | sed 's/\s*[0-9]*\s*//'`
+dir=`git rev-parse --show-prefix`
+remote=`git remote get-url origin`
+root=${remote%.git}
+
+cat > package.ini <<FIN
```
Comment (jcbrinfo) [inline bin/nit-makepackageini]: * Please use an english identifier (like `EOF`) or apostrophes.
* Why not `printf` or `echo`?

**Comment by privat** [comment]
reroll: rewrite the tool for more readability. fix nit.git heuristic. use English words to end heredoc
